src/authentication/authentication.ts

```diff
@@ -90,6 +89,9 @@ const organizationsDataApi = new OrganizationsDataApi(
 );
 
 async function onUserLoginRedirect() {
+  if (typeof window === "undefined") {
```

Comment:
What do you think to join these conditions?
```suggestion
  if (
    typeof window === "undefined" ||
    window.location.pathname !== "/session"
  ) {
    return { type: "not-executed" } as const;
  }
```

Reply:
the (typeof window === "undefined") checks if we are SSR phase.
window.location.pathname !== "/session" aborts parsing auth token from query string if we are not on the route dedicated for the third party authenticatino redirect.
i would not mix them. but fine for me accepting your cahnge request. let me know if it is fine for you leave it as it is or you want me to change it as suggested.